src/spectrecoff-cli/commands/BreakdownChart.fs

```diff
@@ -0,0 +1,51 @@
+﻿namespace SpectreCoff.Cli.Commands
+
+open Spectre.Console.Cli
+open Spectre.Console
+open SpectreCoff.Cli
+open SpectreCoff.Output
+open SpectreCoff.Rule
+open SpectreCoff.Layout
+open SpectreCoff.Chart
+open SpectreCoff.Chart.BreakdownChart
+
+type BreakdownChartSettings() =
+    inherit CommandSettings()
+
+type BreakdownChartExample() =
+    inherit Command<BreakdownChartSettings>()
+    interface ICommandLimiter<BreakdownChartSettings>
+
+    override _.Execute(_context, _settings) =
+        Many [
+            NL
+            E "My life as a developer :("
+            NL
+        ] |> toConsole
+
+        let items = [
+            ChartItem ("Refinements", 2)
+            ChartItem ("Dailies", 2)
+            ChartItem ("Retrospectives", 1)
+            ChartItem ("Random meetings", 7)
+            ChartItem ("Fixing bugs", 3)
+            ChartItemWithColor ("Developing features", 1, Color.Red)
+        ]
+
+        breakdownChart items |> toConsole
+        0
+
+type BreakdownChartDocumentation() =
+    inherit Command<BreakdownChartSettings>()
+    interface ICommandLimiter<BreakdownChartSettings>
+
+    override _.Execute(_context, _settings) =
+        Theme.setDocumentationStyle
+        Many [
+            NL
+            pumped "Breakdown Chart" |> alignedRule Left
```

Comment:
Would not mix `pumped `and `E` -> P, E or both in long form?

Reply:
Unless I misunderstand something here, there is no abbreviated version of `pumped`. 
It's the function that creates the markup string, not the `OutputPayload`, and I think it is correct for `alignedRule` to expect a string.
For the record, `edgy`, `pumped` and `calm` still make me chuckle.

Reply:
You're right, sry. Hehe.